### ISSUE TITLE
Improper Acessebility

### ISSUE THREAD

**Issue by TitanNano** [PROGRAP/sw6-google-customer-reviews#16]
The Badge is currently lacking proper accessibility information. There is no alt attribute for the Google logo, and the star rating should have a role of `meter` to correctly communicate its function.
